WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteAdapter.kt

```diff
@@ -16,8 +16,8 @@ import org.wordpress.android.ui.mysite.MySiteCardAndItem.Item.ListItem
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.CATEGORY_HEADER_ITEM
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.DOMAIN_REGISTRATION_CARD
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.LIST_ITEM
-import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.POST_CARD_CREATE_FIRST
-import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.POST_CARD_DRAFT_OR_SCHEDULED
+import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.POST_CARD_WITHOUT_POST_ITEMS
+import org.wordpress.android.ui.mysite.MySiteCardAndItem.Type.POST_CARD_WITH_POST_ITEMS
```

Comment:
Minor (🔍): Consider optimizing the import and use the full path, as if `MySiteCardAndItem.Type.SITE_INFO_CARD.ordinal` instead of `SITE_INFO_CARD.ordinal` within `onCreateViewHolder(...)`.

Reply:
In 1116d34